bpmodify/bpmodify.go

```diff
@@ -240,7 +238,7 @@ func processParameter(value parser.Expression, paramName, moduleName string,
 		modified = modified || m
 	}
 
-	if (wasSorted || *sortLists) && modified {
```

Comment:
I hadn't realized this defaults to only operating on already-sorted lists.  I don't understand why this is in issue for you.  Why is a list with version numbers different?  Would switching to numerical sort solve  your problem?

Reply:
>  I don't understand why this is in issue for you. Why is a list with version numbers different? 
adding "10" to `versions: ["1", "2"]` resulting `versions: ["1", "10", "2"]`.

> Would switching to numerical sort solve your problem?
Yes, that's my original trial. (Adding `-n` to do numerical sort). But I found that passing `-s` explicitly to sort is better. Do you want to keep the original behavior (sorting already-sorted list)?